submissions/Writerman/my_pop_up/css/style.css

```
@@ -0,0 +1,250 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline:none;
+}
+html {
+  height: 100%;
+}
+body {
+  display: grid;
+  min-height: 100%;
+  grid-template-rows: auto 1fr;
+  font-family: sans-serif;
+  font-size: 16px;
+  color: #444;
+}
+.nav {
+  padding: 20px;
+  position: absolute;
+  top: 3.0em;
+  right: 2em;
+  width: 350px;
+  border: 2px solid #666666;
```

```suggestion
  border: 2px solid #666;
```
If you use in other places short syntax, use it everywhere ;)